### ISSUE TITLE
Create s Dfs0123u file such as Dfs2 file with TimeAxisType of CalendarNonEquidistant. 

### ISSUE THREAD

**Issue by mohm-dhi** [DHI/mikeio#154]
There are many cases that the input data are not available in the Equidistant datetime so we cannot use the start_time, dt parameters to create the dfs2 file. I recall that I could create nonequidistant dfs2 file before by providing the datetime values as a parameter but it seems like it is not working anymore. could you please add this feature to "Write" method again. 

Thank you,

**Comment by ecomodeller** [comment]
https://github.com/DHI/mikeio/issues/118#issuecomment-721344596

**Comment by mohm-dhi** [comment]
I hope to did not close this issue because the comments that you send me does not solve the problem. It is good idea to make interpolation and create the Equidistant TimeAxis but there are situations that we do not want that. For example I have observed data of Chlorophyll-a that the sampling did not occurred on a regular basis and this data is just for visualization and model calibration not input for model so it is not a good idea to make them Equidistant by interpolation. To solve this I already added the CalendarNonEquidistant feature to the dfs1 and dfs2 file format in https://github.com/DHI/mikeio/tree/dfs2_NonEqCalendarAxis 
I will make a PR for you to review. 

Thanks,
